### ISSUE TITLE
Use Vega-Embed

### ISSUE THREAD

**Issue by domoritz** [8/vega-preview#2]
With https://github.com/vega/vega-embed, you get tooltips and themes.

**Comment by g3o2** [comment]
And interaction?

**Comment by domoritz** [comment]
Yes, you also get the action buttons with links to the editor and to download charts as images.
